tests/e2e/tests/test_intelligent_oracle_factory.py

```
@@ -0,0 +1,225 @@
+        ),
+        pytest.param(
+            {
+                "chain": testnet_asimov,
+                "account_kwargs": {"account_private_key": account_private_key_1},
+                "contract_address_path": ["tx_data_decoded", "contract_address"],
+            },
+            marks=pytest.mark.testnet,
+        ),
+    ],
+)
+def test_intelligent_oracle_factory_pattern(chain_config):
```

Pylint (3.3.7)

[refactor] Too many local variables (26/15)

(R0914)